### PR TITLE
src/pybind/mgr/snap_schedule: fix non-existent fs traceback error 

### PR DESCRIPTION
**Pull request by gokceni**
Fixes: https://tracker.ceph.com/issues/51630
Signed-off-by: Gokcen Iskender gokceng@gmail.com

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by gokceni** [comment]
jenkins restest this please

**Comment by ljflores** [comment]
```
mypy run-test-pre: PYTHONHASHSEED='2887244939'
mypy run-test: commands[0] | mypy --config-file=../../mypy.ini -m alerts -m balancer -m cephadm -m crash -m dashboard -m devicehealth -m diskprediction_local -m hello -m influx -m iostat -m localpool -m mds_autoscaler -m mgr_module -m mgr_util -m mirroring -m nfs -m orchestrator -m pg_autoscaler -m progress -m prometheus -m rbd_support -m rook -m snap_schedule -m selftest -m stats -m status -m telegraf -m telemetry -m test_orchestrator -m volumes -m zabbix
snap_schedule/module.py:10: note: In module imported here,
snap_schedule/__init__.py:9: note: ... from here:
snap_schedule/fs/schedule_client.py: note: In member "get_schedule_db" of class "SnapSchedClient":
snap_schedule/fs/schedule_client.py:134: error: Name "false" is not defined
snap_schedule/fs/schedule_client.py:147: error: Argument 2 to "open_ioctx" has incompatible type "Optional[str]"; expected "Union[int, str]"
Found 2 errors in 1 file (checked 31 source files)
```

File: src/pybind/mgr/snap_schedule/fs/schedule_client.py [hunk 1]
```diff
@@ -131,7 +131,10 @@ def dump_on_update(self) -> None:
     def get_schedule_db(self, fs: str) -> sqlite3.Connection:
         if fs not in self.sqlite_connections:
             poolid = self.get_metadata_pool(fs)
-            assert poolid, f'fs "{fs}" not found'
+            if poolid == false:
```
Comment (ideepika) [inline src/pybind/mgr/snap_schedule/fs/schedule_client.py:134]: if not poolid:

**Comment by ideepika** [comment]
might want to fix locally: 
`snap_schedule/module.py:10: in <module>
    from .fs.schedule_client import SnapSchedClient
E     File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/snap_schedule/fs/schedule_client.py", line 134
E       if not poolid:
E                    ^
E   IndentationError: unindent does not match any outer indentation level`

File: src/ceph.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/java/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/java/test" isTestSource="true" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (batrick) [inline src/ceph.iml:12]: There seems to be an extraneous commit here and this file is from an IDE? Please undo this change.

**Review by batrick — CHANGES_REQUESTED** [review]

**Comment by ideepika** [comment]
> @ideepika does this one need an approval? Or is there still something wrong with it?

synced with @varshar16 earlier, was waiting on someone from FS team to review, thanks @batrick @ljflores for getting back to it!

**Comment by vshankar** [comment]
@gokceni ping?

**Comment by gokceni** [comment]
@vshankar I deleted the iml file. Please feel free to take over this PR since I will not be available for some time

**Review by vshankar — APPROVED** [review]

**Comment by vshankar** [comment]
@gokceni please rebase.

**Pull request closed without merging**